Sources/FluidAudio/TextToSpeech/TtsManager.swift

```diff
@@ -0,0 +1,217 @@
+import Foundation
+import OSLog
+
+@available(macOS 13.0, *)
+public final class TtSManager {
+
+    private let logger = AppLogger(subsystem: "com.fluidaudio.tts", category: "TtSManager")
+
+    private var ttsModels: TtsModels?
+    private var isInitialized = false
+
+    public init() {}
+
+    public var isAvailable: Bool {
+        isInitialized
+    }
+
+    public func initialize(models: TtsModels) async throws {
+        logger.info("Initializing TtSManager with provided models")
+
+        self.ttsModels = models
+
+        await KokoroModelCache.shared.registerPreloadedModels(models)
```

Comment:
note - for Swift 6, we need to avoid `shared` as much as possible. Its a pain in teh ass to migrate later

Reply:
BUt also why do we need Model cache - we dont do that for other models